### PR TITLE
Enhacements packages `yiisoft/form`.

### PR DESCRIPTION
**Pull request by terabytesoftw**
| Q             | A
| ------------- | ---
| Is bugfix?    | ✔️
| New feature?  | ❌
| Breaks BC?    | ❌

- [x] Clean up all code.
- [x] Fix create `FormErrors` custom class.
- [x] Remove code duplicate. 
- [x] Rename `CommonAttributes` for `InputAttributes`.
- [x] Change `assertEquals` for `assertSame`.
- [x] Add more tests Code Coverage 100%.

### CODE REVIEW

File: src/FormModel.php [hunk 1]
```diff
@@ -26,17 +26,14 @@
...
     private array $attributes;
-    /** @psalm-var array<string, array<array-key, string>> */
-    private array $attributesErrors = [];
-    private string $formErrorsClass = FormErrors::class;
     private FormErrorsInterface $formErrors;
     private ?Inflector $inflector = null;
     private bool $validated = false;
 
-    public function __construct()
+    public function __construct(string $formErrorsClass = FormErrors::class)
```
Comment (samdark) [inline src/FormModel.php:33]: I'll need to accept/pass this argument in every form model I create. Is it possible to pass it with a method later?

**Review by samdark — APPROVED** [review]

**Review by samdark — CHANGES_REQUESTED** [review]
Is it possible to keep constructor clean?

**Comment by terabytesoftw** [comment]
> Is it possible to keep constructor clean?

Yes tomorrow I'll do it with a setter.

File: src/FormModel.php (hunk 1 above)
Comment (terabytesoftw) [reply src/FormModel.php]: fixed.

**Comment by terabytesoftw** [comment]
No merge.

**Comment by terabytesoftw** [comment]
Closed in favor https://github.com/yiisoft/form/pull/160

**Pull request closed without merging**